pkg/resources/crd_helpers.go

```diff
@@ -26,6 +26,157 @@ func BundleCrd() *apiext_v1b1.CustomResourceDefinition {
 	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md
 	// https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/namespaces.md
 	// https://github.com/kubernetes/kubernetes/tree/master/api/openapi-spec
+
+	// definitions are not supported, do what we can :)
```

Comment:
Extracted each definition into a variable and use it instead of a reference to that definition.